src/IoC/IoC.java

```diff
@@ -0,0 +1,19 @@
+package IoC;
+
+import java.util.Map;
+
+public class IoC {
```

Comment:
You need a way to register the instances

Reply:
You cannot register the same instance twice

Reply:
Can you explain a little further? Becase im using the singleton pattern, there's only one instance of the IoC class to be used

Reply:
the instances that you are going to store in the IoC needs to to be registered some how. Create a way to allow them to be registered

Reply:
There will be 2 Enum classes with the string name of all the services and repositories, and in the constructor of the IoC i'll add the values and instances in the map, is that what you mean?

Reply:
> `IoC` is nopt a very common java convention. Rename to `Ioc` for both folder and class.

solved

Reply:
> No. I want to be able to do:
> 
> ```
> Ioc.getInstance().register("some-key", "some-value");
> ```

solved